package.json

```diff
@@ -37,6 +37,7 @@
     "@rainbow-me/rainbowkit": "^2.1.3",
     "@tanstack/react-query": "^5",
     "clsx": "^2.1.1",
+    "geist": "^1.3.1",
```

Comment:
I don't love adding a new dependency but I think it's the only way to support this theme